engine/src/exec-interface2.cpp

```diff
@@ -2395,36 +2397,25 @@ void MCInterfaceEvalFocusedObjectAsObject(MCExecContext& ctxt, MCObjectPtr& r_ob
     ctxt . LegacyThrow(EE_CHUNK_NOTARGET);
 }
 
-static MCStack *MCInterfaceTryToEvalBinaryStack(MCStringRef p_data, bool& r_binary_fail)
+MCStack *MCInterfaceTryToEvalStackFromString(MCStringRef p_data)
 {
-    uint4 offset;
-    MCStack *t_stack;
-    bool t_binary_fail;
+    char_t* t_string;
+    uindex_t t_length;
+    if (!MCStringConvertToNative(p_data, t_string, t_length))
```

Comment:
...unless `MCS_fakeopen()` takes ownership of the buffer it's passed, of course...